model_zoo/lbann2.cpp

```diff
@@ -183,51 +183,78 @@ model * build_model_from_prototext(int argc, char **argv, lbann_data::LbannPB &p
       pb_model->set_num_parallel_readers(procs_per_model);
     }
 
-    if (master) {
-      std::cout << "Model settings" << std::endl
-                << "  Models              : " << comm->get_num_models() << std::endl
-                << "  Processes per model : " << procs_per_model << std::endl
-                << "  Grid dimensions     : " << comm->get_model_grid().Height() << " x " << comm->get_model_grid().Width() << std::endl;
-      std::cout << std::endl;
-    }
-
     // Save info to file; this includes the complete prototext (with any over-rides
     // from the cmd line) and various other info
     //save_session(comm, argc, argv, pb);
 
-    // Check for cudnn, with user feedback
-    cudnn::cudnn_manager *cudnn = nullptr;
+    // Report useful information
```

Comment:
Duplicate of lbann.cpp?

Reply:
lbann2.cpp, lbann_cycgan.cpp, and lbann_gan.cpp are very similar to lbann.cpp. I know @samadejacobs is working on implementing the GAN stuff within the main LBANN pipeline. We should start thinking about how to replicate the functionality in lbann2.cpp so we can deprecate it.

Reply:
We should at least keep them for now until may be after SplashApp.

Reply:
I've opened #468 to discuss this.